src/meta-srv/src/procedure/region_follower/add_region_follower.rs

```diff
@@ -67,12 +67,26 @@ impl AddRegionFollowerProcedure {
         // loads the datanode peer and check peer is alive
         self.data.peer = self.data.load_datanode_peer(&self.context).await?;
 
-        // loads the datanode table value
-        self.data.datanode_table_value = self.data.load_datanode_table_value(&self.context).await?;
-
         // loads the table route of the region
         self.data.table_route = self.data.load_table_route(&self.context).await?;
+        let region_leader_datanode_id = {
+            let table_route = self.data.physical_table_route().unwrap();
+            table_route
+                .region_routes
+                .iter()
+                .find(|region_route| region_route.region.id == self.data.region_id)
+                .map(|region_route| region_route.leader_peer.as_ref().unwrap().id)
```

Comment:
Is it possible that the leader_peer is none?

Reply:
Resolved in [5fffbe6](https://github.com/GreptimeTeam/greptimedb/pull/5781/commits/5fffbe699f0ff5825d0789b210b51199b64c6ebd)